app/(dashboard)/_components/smart-notes/index.tsx

```
@@ -62,10 +96,15 @@ export default function SmartNotes() {
       }
     } catch (error: any) {
       console.error('Failed to create note:', error);
-      if (error.message.includes('log in')) {
+      if (error.message && error.message.includes('log in')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)